NuGet.Config

```diff
@@ -2,6 +2,6 @@
 <configuration>
   <packageSources>
     <clear />
-	<add key="NuGet" value="https://api.nuget.org/v3/index.json" />
+  <add key="IDDP" value="https://identitydivision.pkgs.visualstudio.com/_packaging/IDDP/nuget/v3/index.json" />
```

Comment:
Our external customers will not be able to reach these packages and their local builds will fail.